### PR TITLE
Virtualbox 6.1 support

### PR DESCRIPTION
**Pull request by rkoster**
Addresses #18 by adding SATA disk controller support.

In addition, Golang has been updated, go modules enabled and blob storage changed to git lfs.

The change to git lfs was made so blob storage costs can be easily be paid for by the CF Foundation.

More details about using git lfs for bosh releases can be found [here](https://www.starkandwayne.com/blog/bosh-releases-with-git-lfs/).

### CODE REVIEW

File: src/bosh-virtualbox-cpi/vm/portdevices/port_device.go [hunk 1]
```diff
@@ -2,18 +2,26 @@ package portdevices
...
+var (
+	// Covers `"SATA-ImageUUID-0-0"="a840e5e0-947c-4e63-ac2f-678a86d13980"`
+	portImagewUUIDConfig = regexp.MustCompile(`^"(\w+)-ImageUUID-(\d+)-(\d+)"="(.+)"$`)
+)
+
 type PortDevice struct {
 	driver driver.Driver
 	vmCID  apiv1.VMCID
 
 	controller string // e.g. scsi, ide
```
Comment (ramonskie) [inline src/bosh-virtualbox-cpi/vm/portdevices/port_device.go:23]: missing sata

**Review by bgandon — CHANGES_REQUESTED** [review]
Overall, I think we should stick to the original blobstore in this PR, even though I understand `git lfs` local blobstore is handy for testing. Let me know if you can't get a hand on the AWS blobstore credentials.

File: jobs/virtualbox_cpi/spec [hunk 2]
```diff
@@ -28,8 +28,8 @@ properties:
     default: "~/.bosh_virtualbox_cpi"
 
   storage_controller:
-    description: "For some reason IDE controller seems to be more reliable. Available: scsi or ide."
-    default: ide
+    description: "For some reason IDE controller seems to be more reliable. Available: scsi, ide or sata."
```
[the remark this answers is not in the document]
Comment (ramonskie) [reply jobs/virtualbox_cpi/spec]: already solved and can be ignored

**Comment by rkoster** [comment]
Trigger cfgit-bot

**Review by bgandon — APPROVED** [review]

**Pull request merged**